imports/ui/components/reader.js

```diff
@@ -30,17 +30,31 @@ Template.reader.onCreated(function(){
 	// i guess it would be better to pull this number
 	// from DB to avoid fatal disconnections
 	instance.data.obj._atIndex = -1
-	// this is where we're going to store all the 
+	// this is where we're going to store all the
 	// qcm answers of players
 	instance.data.obj.answered = []
-	// we need to store what loot the players have
+  // we need to store what loot the players have
 	// gained in order to modify the dice rolls they are
 	// going to do at the end.
 	instance.data.obj.modifiers = []
+
+
+Template.reader.onRendered(function(){
+
+		setTimeout(function(){
```

Comment:
this should be moved to line 616 in function loadAnswer

Reply:
i did exactly that.